src/nupic/proto/ConnectionsProto.capnp

```diff
@@ -1,43 +1,24 @@
 @0xb1b8a459d70716ad;
 
-# Next ID: 5
+# Next ID: 2
```

Comment:
OK but this is a breaking change to anything that's saved a connections instance.